Mathlib/AlgebraicGeometry/EllipticCurve/NormalForms.lean

```diff
@@ -0,0 +1,106 @@
+/-
+Copyright (c) 2024 Jz Pan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jz Pan
+-/
+import Mathlib.AlgebraicGeometry.EllipticCurve.Weierstrass
+
+/-!
+
+# Some normal forms of elliptic curves
+
+This file defines some normal forms of Weierstrass equations of elliptic curves.
+
+## Main definitions
+
+- ...
+
+## References
+
+* [J Silverman, *The Arithmetic of Elliptic Curves*][silverman2009]
+
+## Tags
+
+elliptic curve, weierstrass equation, normal form
+
+-/
+
+universe u
+
+variable {R : Type u}
+
+namespace WeierstrassCurve
+
+variable (W : WeierstrassCurve R)
+
+/-! ### Normal forms of characteristic ≠ 2
+-/
+
+/-- A `WeierstrassCurve` is of normal form of characteristic ≠ 2, if its a₁ and a₃ are zero.
```

Comment:
```suggestion
/-- A `WeierstrassCurve` is of normal form of characteristic ≠ 2, if its `a₁` and `a₃` coefficients are zero.
```

Reply:
Also: should it be "in normal form" instead of "of normal form"?

Reply:
Thanks! Fixed.